tests/e2e/p/permissionless_subnets.go

```diff
@@ -159,7 +159,7 @@ var _ = e2e.DescribePChain("[Permissionless Subnets]", func() {
 						Validator: txs.Validator{
 							NodeID: validatorID,
 							Start:  uint64(validatorStartTime.Unix()),
-							End:    uint64(validatorStartTime.Add(5 * time.Second).Unix()),
+							End:    uint64(validatorStartTime.Add(time.Minute).Unix()),
```

Comment:
Well actually I modified the endTime here. THe point of dropping pending stakers is that startTime becomes a placeholder.
In a nutshell:

1. What will matter in a tx will be EndTime-StartTime, not EndTime and StartTIme by themselves
2. StartTime will be really the chain time when the transaction is accepted

A drawback of this solution is that a delegator won't be able anymore to fully cover its validator lifetime (since it has no control over start time). Here I slightly reduce the EndTime (i.e. the staking duration really) so to make sure that delegator will not outlive its delegator and it will be accepted then (delegators outliving their validators are rejected).